### PR TITLE
Remove the video device.

### PR DESCRIPTION
**Pull request by mykaul**
I don't see why we need VNC. We get fine with SSH and console.

### CODE REVIEW

File: lago/vm.py [hunk 1]
```diff
@@ -266,9 +266,7 @@ def extract_paths(self, paths, ignore_nopath):
 
     @_check_defined
     def vnc_port(self):
```
Comment (nvgoldin) [inline lago/vm.py]: As this drops entirely VNC support, I think its better to leave no trace of it(remove this function and others that relay on it)

**Comment by mykaul** [comment]
Dropped all mentioning of it, per the request above.

**Comment by nvgoldin** [comment]
ci merge please

**Comment by nvgoldin** [comment]
Looks like libguestfs failure - doesn't seem related as one of the builds went ok. Although wondering why it used libguestfs here(means ssh was not reachable..).
```
20:09:17 # Error while running thread
20:09:17 # Traceback (most recent call last):
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/utils.py", line 55, in _ret_via_queue
20:09:17 #     queue.put({'return': func()})
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/prefix.py", line 1108, in _collect_artifacts
20:09:17 #     vm.collect_artifacts(path, ignore_nopath)
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/plugins/vm.py", line 496, in collect_artifacts
20:09:17 #     ignore_nopath=ignore_nopath
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/plugins/vm.py", line 311, in extract_paths
20:09:17 #     return self.provider.extract_paths(paths, *args, **kwargs)
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/vm.py", line 262, in extract_paths
20:09:17 #     self._extract_paths_gfs(paths=paths, ignore_nopath=ignore_nopath)
20:09:17 #   File "/usr/lib/python2.7/site-packages/lago/vm.py", line 517, in _extract_paths_gfs
20:09:17 #     gfs_cli.mount_ro(rootfs, '/')
20:09:17 #   File "/usr/lib64/python2.7/site-packages/guestfs.py", line 7414, in mount_ro
20:09:17 #     r = libguestfsmod.mount_ro(self._o, mountable, mountpoint)
20:09:17 # RuntimeError: mount_ro: /dev/sda3 on / (options: 'ro'): mount: /dev/sda3: can't read superblock
```

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**